mail/postfix/Makefile

```diff
@@ -177,136 +181,78 @@ newaliases_path=/usr/bin/newaliases
 mailq_path=/usr/bin/mailq
 shlib_directory=/usr/lib/postfix
 meta_directory=/etc/postfix
+# postinit script may override the values below if the external
+# drive is at a mountpoint other than /data/
```

Comment:
How does it happen exactly? Couldn't understand this comment, sorry

Reply:
While there is no convention about the mount point for persistent data in LEDE/OpenWrt, IMO we should do some heuristics to identify the most appropriate location for the Postfix data files during the initial installation:
* the postinit script should check if an external storage device is mounted
* if an external device is found, use that device's mount point
* if extroot is used, use /data/

The comments "postinit script may override..." is a placeholder for future work. At this time, I don't know how to properly detect if extroot is used. I can remove lines 184 and 185 if they look confusing.

Reply:
Ok, but could you please rephrase it to indicate it is not implemented, or just remove it, because it might look confusing.

Reply:
Point taken, will rephrase.